buster/retriever/pickle.py

```diff
@@ -0,0 +1,25 @@
+import pandas as pd
+
+from buster.retriever.base import Retriever
+
+
+class PickleRetriever(Retriever):
+    def __init__(self, filepath: str):
+        self.filepath = filepath
+        self.documents = pd.read_pickle(filepath)
+
+    def get_documents(self, source: str) -> pd.DataFrame:
+        if self.documents is None:
+            raise FileNotFoundError(f"No documents found at {self.filepath}. Are you sure this is the correct path?")
+
+        documents = self.documents.copy()
+        if "current" in documents.columns:
```

Comment:
whats supposed to be in the `"current"` column? can we add a comment on that?

Reply:
just a 1 or 0, when there are multiple versions of a document. Mostly obsolete, but requires some change elsewhere to be properly removed, so it should go in another PR

Reply:
ok we should reference this in an issue somehow

Reply:
referenced in #78